Assets/Xbox Live/Scripts/UserProfile.cs

```diff
@@ -206,31 +207,78 @@ public IEnumerator SignInAsync()
         // Throw any exceptions if needed.
         if (signInStatus == SignInStatus.Success)
         {
+            if (!subscribedToEventHandler)
```

Comment:
I'm confused why you are subscribing to this event differently depending on whether the user is already signed in or not.  It seems like we care either way.  Could you just subscribe to this event once in Start() and get rid of the subscribedToEventHandler variable?

Reply:
Sure